app/app.go

```diff
@@ -466,12 +472,18 @@ func New(
 		app.OracleKeeper,
 		cast.ToBool(appOpts.Get(leveragetypes.FlagEnableLiquidatorQuery)),
 	)
-	app.LeverageKeeper = *app.LeverageKeeper.SetHooks(
-		leveragetypes.NewMultiHooks(
-			app.OracleKeeper.Hooks(),
-		),
+	app.IncentiveKeeper = incentivekeeper.NewKeeper(
+		appCodec,
+		keys[incentive.StoreKey],
+		app.BankKeeper,
+		app.LeverageKeeper,
+	)
+	app.LeverageKeeper = *app.LeverageKeeper.SetTokenHooks(
```

Comment:
I see comments in app.go along the lines of

> // IBC Keeper must be a pointer in the app, so we can SetRouter on it correctly

and

> // NOTE: stakingKeeper above is passed by reference, so that it will contain these hooks

I will need to verify that leverage hooks are being set successfully in app after making this change